keps/0023-enable-disable-features.md

```diff
@@ -0,0 +1,131 @@
+---
+kep-number: 23
+title: Enabling and Disabling features in KUDO Operators
+authors:
+  - "@zmalik"
+owners:
+  - "@zmalik"
+creation-date: 2020-01-28
+last-updated: 2020-01-28
+status: provisional
+---
+
+# Enabling and Disabling features in KUDO Operators
+
+## Summary 
+
+KUDO enables Operator developers to create Operators and develop features for them. 
```

Comment:
maybe it's rather exposing features of the underlying applications?

Reply:
SGTM
I just didn't want to use words like `extend operators` which would diverge from the word `feature`